tests/core/data_layer/test_data_rpc.py

```diff
@@ -803,7 +803,11 @@ async def offer_setup_fixture(
 
 async def populate_offer_setup(offer_setup: OfferSetup, count: int) -> OfferSetup:
     if count > 0:
-        for store_setup, value_prefix in {offer_setup.maker: b"\x01", offer_setup.taker: b"\x02"}.items():
+        setups: Tuple[Tuple[StoreSetup, bytes], Tuple[StoreSetup, bytes]] = (
```

Comment:
This doesn't look related here?

Reply:
Hum...  I thought mypy was complaining or something.  But maybe I was just annoyed it wasn't detecting the type or...  Yep, I'll make a separate pr for this.

https://github.com/chia-network/chia-blockchain/commit/680d2833e4965968c17be673f2d7d67fc0ea676d
https://github.com/Chia-Network/chia-blockchain/pull/16257

Reply:
Oh yeah, this is because `DataLayer` is no longer hashable so we need to avoid hashing it such as as a dict key.